src/webcomponents/organization/admin/user-admin-password-change.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * Copyright 2015-2024 OpenCB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {html, LitElement} from "lit";
+import UtilsNew from "../../../core/utils-new.js";
+import LitUtils from "../../commons/utils/lit-utils.js";
+import NotificationUtils from "../../commons/utils/notification-utils.js";
+import "../../user/user-password-change.js";
+
+export default class UserAdminPasswordChange extends LitElement {
+
+    constructor() {
+        super();
+
+        this.#init();
+    }
+
+    createRenderRoot() {
+        return this;
+    }
+
+    static get properties() {
+        return {
+            userId: {
+                type: String
+            },
+            organization: {
+                type: Object,
+            },
+            opencgaSession: {
+                type: Object
+            },
+            displayConfig: {
+                type: Object
+            },
+        };
+    }
+
+    #init() {
+        this.user = {}; // Original object
+        this._user = {}; // Updated object
+        this.userId = "";
+        this.displayConfig = {};
+        this.updatedFields = {};
+
+        this._config = this.getDefaultConfig();
+    }
+
+    #setLoading(value) {
+        this.isLoading = value;
+        this.requestUpdate();
+    }
+
+    #initConfigNotification() {
+        this._config.notification = {
+            title: "",
+            text: "Some changes have been done in the form. Not saved changes will be lost",
+            type: "notification",
+            display: {
+                visible: () => {
+                    return UtilsNew.isNotEmpty(this.updatedFields);
+                },
+                notificationType: "warning",
+            },
+        };
+    }
+
+    #initOriginalObjects() {
+        this._user = UtilsNew.objectClone(this.user);
+        this.updatedFields = {};
+    }
+
+    update(changedProperties) {
+        if (changedProperties.has("userId")) {
+            this.userIdObserver();
+        }
+        if (changedProperties.has("displayConfig")) {
+            this._config = this.getDefaultConfig();
+            if (!this._config?.notification) {
+                this.#initConfigNotification();
+            }
+        }
```

Comment:
The `displayConfig` is not used, why do you need it?

Reply:
This component is not in use yet and its use has been commented out. I will work on it if we need it.